tests/integration/targets/test_mysql_user/tasks/issue-265.yml

```
@@ -0,0 +1,168 @@
+      mysql_user:
+        <<: *mysql_params
+        name: '{{ user_name_1 }}'
+        password: '{{ user_password_1 }}'
+        state: present
+        force_context: yes
+      register: result
+
+    - name: assert output message mysql user was created
+      assert:
+        that:
+          - "result.changed == true"
+
+    - include: assert_user.yml user_name={{user_name_1}}
+
+    # Test user removal
+    - name: remove mysql user {{user_name_1}}
+      mysql_user:
+        <<: *mysql_params
+        name: '{{user_name_1}}'
+        password: '{{user_password_1}}'
+        state: absent
+        force_context: yes
+      register: result
+
+    - name: assert output message mysql user was removed
+      assert:
+        that:
+          - "result.changed == true"
+
+    # Test blank user removal
+    - name: create blank mysql user to be removed later
+      mysql_user:
+        <<: *mysql_params
+        name: ""
+        state: present
+        force_context: yes
+        password: 'KJFDY&D*Sfuydsgf'
+
+    - name: remove blank mysql user with hosts=all (expect changed)
+      mysql_user:
+        <<: *mysql_params
+        user: ""
+        host_all: true
+        state: absent
+        force_context: yes
+      register: result
+
+    - name: assert changed is true for removing all blank users
+      assert:
+        that:
+          - "result.changed == true"
+
+    - name: remove blank mysql user with hosts=all (expect ok)
+      mysql_user:
+        <<: *mysql_params
+        user: ""
+        host_all: true
+        force_context: yes
+        state: absent
+      register: result
+
+    - name: assert changed is true for removing all blank users
+      assert:
+        that:
+          - "result.changed == false"
+
+    - include: assert_no_user.yml user_name={{user_name_1}}
+
+    # Tests with force_context: no
+    # Test user creation
+    - name: Drop mysql user if exists
+      mysql_user:
+        <<: *mysql_params
+        name: '{{ user_name_1 }}'
+        state: absent
+      ignore_errors: yes
+
+    # Tests with force_context: yes
+    # Test user creation
+    - name: create mysql user {{user_name_1}}
+      mysql_user:
+        <<: *mysql_params
+        name: '{{ user_name_1 }}'
+        password: '{{ user_password_1 }}'
+        state: present
+        force_context: yes
+      register: result
+
+    - name: assert output message mysql user was created
+      assert:
+        that:
+          - "result.changed == true"
+
+    - include: assert_user.yml user_name={{user_name_1}}
+
+    # Test user removal
+    - name: remove mysql user {{user_name_1}}
+      mysql_user:
+        <<: *mysql_params
+        name: '{{user_name_1}}'
+        password: '{{user_password_1}}'
+        state: absent
+        force_context: no
+      register: result
+
+    - name: assert output message mysql user was removed
+      assert:
+        that:
+          - "result.changed == true"
```

```suggestion
          - "result.changed == true"
```
No need to change, just another form of the same can be just
```
- result is changed
```
it's a question of taste, i think

---

I prefer the latter in my playbooks but i actually just copied most of that from the other playbooks 😅

---

:)